src/texttransformation.js

```diff
@@ -94,6 +94,21 @@ export default class TextTransformation extends Plugin {
 		} );
 
 		this.editor = editor;
+
```

Comment:
I don't think we need to expose any of these. I don't see why:

* we expose any property on the class,
* we need to keep original transformations and normalized transformations.

Previously, everything happened locally in `_enableTransformationWatchers()` and that was fine. Also, we shouldn't need to keep both structures – that's really confusing now why do you iterate over items in `configuredTransformations` but later use `normalizedConfiguredTransformations`. Why not using only the latter and dropping the former?

Reply:
You're totally right, I overcomplicated this class... Fixed it already!